buildSrc/subprojects/plugins/src/test/kotlin/org/gradle/plugins/performance/DetermineBaselinesTest.kt

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright 2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.plugins.performance
+
+
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.mockkObject
+import io.mockk.mockkStatic
+import org.gradle.api.Action
+import org.gradle.kotlin.dsl.*
+import org.gradle.process.ExecResult
+import org.gradle.testfixtures.ProjectBuilder
+import org.junit.Before
+import org.junit.Test
+
+
+class DetermineBaselinesTest {
+    val project = ProjectBuilder.builder().build()
+
+    @Before
+    fun setUp() {
+        // mock project.execAndGetStdout
+        mockkStatic("org.gradle.kotlin.dsl.Kotlin_dsl_upstream_candidatesKt")
```

Comment:
❓ Doesn't this require a matching `unmockkStatic` in an `@After` method?

Reply:
Yes, there should be a `clearMocks`.